src/components/buttons-group.js

```diff
@@ -1,6 +1,6 @@
-function buttonsGroup(buttonsId,buttons) {		
+function buttonsGroup(divsId,buttons,htmlClass) {		
```

Comment:
why no `groupId` ?

it's the group's id and not div, whats happen if you change `div` to `section`?

Reply:
change all the divs for section? maybe it'll still work

Reply:
No, it was just a exemplo! 

Keep it as div